packages/core/demo/data/line.ts

```diff
@@ -52,7 +52,8 @@ export const sparklineLoadingOptions = {
 			visible: false,
 			mapsTo: "value",
 			title: "Conversion rate",
-			scaleType: "linear"
+			scaleType: "linear",
+			ticks: {values: [16]}
```

Comment:
I think that using the configuration for `values` is confusing here. That configuration was to provide values within the domain that would be visually rendered to the chart, and the sparkline does the opposite - rendering no text ticks. I know its called `ticksValues` **within** sparkline rendering code but it feels confusing to use it in the configurations when it doesn't work the same way as it does for the charts that _do render axis tick text_

Reply:
@natashadecoste 
Thanks for the review comment.

I am not quite sure about what you refer as _render axis tick text_?  `ticks: {values: [16]}` in here renders a horizontal line instead of _axis tick text_. 

And also, according to @theiliad 's previous suggestion, `loading Sparkline` is one kind of `Skeleton` so we are reusing methods from `Skeleton` to make a `loading Sparkline`.
https://github.com/carbon-design-system/carbon-charts/pull/837#pullrequestreview-513799536

Hope this could explain why I use` ticksValues` to render a horizontal loading sparkline.